projects/orangecrab/.gitignore

```
@@ -0,0 +1,8 @@
+01-clash/
+02-hdl/
+03-net/
+04-bitstream/
+_build/
+dist-newstyle/
+.ghc.environment.*
+*.local
```

```suggestion
/_build/
/dist-newstyle/
.ghc.environment.*
*.local
```

When you add Stack support, you should probably extend this file.

I'm looking at the difference between this file and the one for the `simple` project, and I wonder about a bunch of lines in that `.gitignore`. A bunch are clearly caught by `/_build/` but there are some others where I wonder whether those are still good to include in `simple` at all.